### PR TITLE
Dockerised the python commands

### PR DESCRIPTION
**Pull request by sureshhewabi**

### CODE REVIEW

**Comment by sureshhewabi** [comment]
Ok Yasset, I think something went wrong. I think mistakenly I have click the button to merge it. I will have a look
Suresh

File: thermo-convert-nf/runMetadataPipeline.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ METADATA_DIR=$4
 PROFILE=$5
 MODE=$6
 FILE_LOCATION=$7
+ENABLE_FTP_DOWNLOAD=$8
```
Comment (ypriverol) [inline thermo-convert-nf/runMetadataPipeline.sh]: Can we call this script: lunchLSFConversion.sh

File: thermo-convert-nf/thermo-convert-nf.nf [hunk 2]
```diff
@@ -36,10 +37,14 @@ log.info """\
  Project Accession  : ${params.px_accession}
  Metadata Files     : ${params.metadata_path}
  Submission Mode    : ${params.mode}
+ Raw Files Location : ${params.files_location}
+ Enable FTP download: ${params.ftp_download}
  """
 
 process downloadFiles {
 
+    container 'sureshhewabi/pride-py:0.0.8'
```
Comment (ypriverol) [inline thermo-convert-nf/thermo-convert-nf.nf]: All the PRIDE containers should be deposited here: https://quay.io/organization/pride not in personal namespaces. Try to create an account with quay.io and I will add you to the organization.

File: thermo-convert-nf/runMetadataPipeline.sh (hunk 1 above)
Comment (sureshhewabi) [reply thermo-convert-nf/runMetadataPipeline.sh]: done

File: thermo-convert-nf/thermo-convert-nf.nf (hunk 2 above)
Comment (sureshhewabi) [reply thermo-convert-nf/thermo-convert-nf.nf]: changed to quay.io/pride/pride-py:0.0.8

**Pull request merged**